package.json

```diff
@@ -47,14 +47,15 @@
     "mofo-style": "latest",
     "ncp": "^2.0.0",
     "npm": "^2.7.4",
-    "npm-run-all": "^1.2.2",
+    "npm-run-all": "git://github.com/k88hudson/npm-run-all.git#colors",
```

Comment:
Still working on getting this one to work but haven't had time, which is why this PR has been sitting around